cli.js

```diff
@@ -67,6 +67,8 @@ lighthouse({
   } else {
     prettyPrint(results);
   }
+}).catch(function () {
+  process.exit();
```

Comment:
Hmm, I have not experience the process hanging -- Do you have any steps to reproduce on your side?